gdbinit.py

```diff
@@ -34,7 +42,13 @@ def calculate_hash(file_path):
 
 
 def run_poetry_install(dev=False):
-    command = ["poetry", "install"]
+    check_poetry()  # Check if poetry executable exists in PATH
+    poetry_path = shutil.which("poetry")
+    if poetry_path is None:
+        print("Poetry is not installed. Please install it and try again.")
```

Comment:
```suggestion
        print("Poetry was not found on the $PATH. Please ensure it is installed and on the path, or run `./setup.sh` to update Python dependencies.")
```

Reply:
Just wanted to confirm, if we hit this code path where poetry is not installed or on the path, and then the user manually runs `./setup.sh`, this message should go away until the next time the `poetry.lock` file changes? This is what I want to happen, and I think it'll happen based on reading the code, but if you could test it and confirm that would be great 🙂

Reply:
The above message would only pop if the poetry was not found on the system. If the `poetry.lock` changes, it won't pop the above message unless the user has uninstalled poetry.